### DISCUSSION TITLE
How do you do multiple `lighthouse` runs in CI?

### DISCUSSION THREAD

**Discussion by hirehamir** [GoogleChrome/lighthouse-ci#1053] [Q&A]
I'd like to do one run with whatever settings are in `.lighthouserc.json`, and then another run with the settings in `.lighthouserc.json`, plus another run with added `"chromeFlags"`.

**Comment by hirehamir** [comment]
What I tried at https://github.com/hamirmahal/extract-nutrition-info/commit/eb47b84bd8b9ccc3df69d3dbc15b64d9b04eef48, with 2 `.github/workflows/lighthouse` files, didn't work.

https://github.com/hamirmahal/extract-nutrition-info/actions/runs/9723393020/job/26838464277

**Comment by alexisLefebvre** [comment]
How do you provide the `chromeFlags` parameter to `lhci`?

In theory, you should be able to define 2 jobs on your CI: one without and one with `chromeFlags`.

Or use 2 different configuration files by using something like `--config=./.lighthouserc.json` and  `--config=./.lighthouserc-chromeFlags.json`: https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#configuration-file

**Reply by hirehamir** [reply]
So I tried defining 2 jobs in CI, but I still get 1 Lighthouse report.

https://github.com/hamirmahal/extract-nutrition-info/tree/ci/run-lighthouse-with-night-theme-active

I'd like to get 2 Lighthouse reports, ideally, if that's possible.

![image](https://github.com/user-attachments/assets/11e669ee-6c85-4f7b-865c-d37aba88ccd3)

**Reply by alexisLefebvre** [reply]
I see 2 different reports generated:

- https://github.com/hamirmahal/extract-nutrition-info/actions/runs/10933524796/job/30352136191#step:7:8
- https://github.com/hamirmahal/extract-nutrition-info/actions/runs/10933524794/job/30352136178#step:7:8

**Reply by hirehamir** [reply]
Hmm... Those 2 reports duplicate the same config and settings.

When I try to do 2 different configurations, there are issues.

- https://github.com/hamirmahal/extract-nutrition-info/actions/runs/10933524796/job/30352136408
- https://github.com/hamirmahal/extract-nutrition-info/actions/runs/10933524796/job/30352136191
